lib/rules.js

```diff
@@ -76,6 +76,21 @@ function getHandler(ruleId) {
 }
 
 /**
+ * Get an object with all currently loaded rules
+ * @returns {Object} All loaded rules
+ */
+function getAllLoadedRules() {
+    const allRules = {};
```

Comment:
I'd like to use a `Map` here instead of an object, as it's more indicative of what the return value actually is.